### PR TITLE
Added test for Packages indexes merge

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #87 
Added more tests for `MultiDebian.MergedPackages` class

### CODE REVIEW

File: src/test/java/com/artipie/debian/MultiDebianTest.java [hunk 1]
```diff
@@ -64,6 +62,80 @@ void mergesPackages() throws IOException {
...
+        final String three = this.abcPackageInfo().replace("0.1", "0.3");
+        new MultiDebian.MergedPackages().merge(
+            new ListOf<InputStream>(
+                new ByteArrayInputStream(
+                    new GzArchive().compress(
+                        String.join(
+                            "\n\n", two, this.abcPackageInfo()
+                        ).getBytes(StandardCharsets.UTF_8)
+                    )
+                ),
```
Comment (genryxy) [inline src/test/java/com/artipie/debian/MultiDebianTest.java:119]: @olenagerasimova I'd suggest to extract method for creating a stream from string values. 
For example,  `stream(String... entries)`

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one suggestion

**Comment by olenagerasimova** [comment]
@genryxy thanks, extracted

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Comment by olenagerasimova** [comment]
@rultor release, tag=`0.7`

**Pull request merged**